agent/base

```diff
@@ -9,13 +9,17 @@
 # will catch any ssh failure
 set -o pipefail
 
-if command -v python3 > /dev/null 2>&1 ;then
-    # All python programs/scripts have headers using "python3" by default.
-    :
-elif command -v scl_source > /dev/null 2>&1 ;then
-    # If we don't have python3, but we do have SCL, we only support
-    # Python 3.6.
+if command -v scl_source > /dev/null 2>&1 ;then
```

Comment:
On my RHEL 8 laptop, `command -v scl_source` returns 0. I have no scl collections installed, though. Would it make more sense to check that `scl --list` doesn't report `rh-python36`? (Of course, on RHEL8, `scl --list` is reported as deprecated, and wants you to use `scl list-collections`, which doesn't exist on RHEL7, so scl doesn't make anything easy.)

Or should we instead check that it's really RHEL 7 via a grep of `/etc/redhat-release` or some such, rather than relying on `scl` artifacts?

Reply:
Probably check for RHEL7 and/or the rh-python3x package: scl may be installed but rh-python3x is probably not on non-RHEL7 systems.